injector/src/test/java/edu/ucr/cs/riple/injector/ComprehensiveTest.java

```diff
@@ -68,27 +68,21 @@ public void addFieldMethodParamTest() {
             "import javax.annotation.Nullable;",
             "public class Test {",
             "   Field foo1;",
-            "   @Nullable",
-            "   Field foo2;",
+            "   @Nullable Field foo2;",
```

Comment:
OTOH, fields are probably better annotated in the same line unless they already have a large collection of annotations...